src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAO.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.implicitConversions
 
 class GraphWorkspaceDAO extends WorkspaceDAO with GraphDAO {
 
-  override def save(workspace: Workspace, txn: RawlsTransaction): Workspace = txn withGraph { db =>
+  override def save(workspace: Workspace, txn: RawlsTransaction): WorkspaceContext = txn withGraph { db =>
```

Comment:
why does save return workspace context?  i think all our save methods return what you saved